packages/cmf/__tests__/Dispatcher.test.js

```diff
@@ -91,80 +86,86 @@ describe('Testing <Dispatcher />', () => {
 	it('should not prevent event propagation by default', () => {
 		const dispatchActionCreator = jest.fn();
 		const onClick = jest.fn();
-		const wrapper = mount(
-			<div onClick={onClick}>
-				<Dispatcher
-					dispatchActionCreator={dispatchActionCreator}
-					onClick="noOp"
-					onDoubleClick="existingActionCreator:id"
-				>
-					<a />
-				</Dispatcher>
-			</div>,
-			mock.Provider.getEnzymeOption({ registry, onError }),
+		const wrapper = render(
+			<mock.Provider registry={registry}>
+				<div onClick={onClick}>
+					<Dispatcher
+						dispatchActionCreator={dispatchActionCreator}
+						onClick="noOp"
+						onDoubleClick="existingActionCreator:id"
+					>
+						<a />
+					</Dispatcher>
+				</div>
+			</mock.Provider>,
 		);
-		wrapper.find('a').simulate('click');
+		fireEvent.click(wrapper.container.querySelector('a'));
```

Comment:
https://testing-library.com/docs/react-testing-library/migrate-from-enzyme#test-2-input-texts-must-have-correct-value

> When you use React Testing Library, you should not try to access the DOM like how you would do by document.querySelector API (which you can incidentally use in your tests, but it's not recommended for the reasons stated in this paragraph).

I would encourage you to use text into the anchor instead